packages/repack/src/webpack/plugins/ChunksToHermesBytecodePlugin/utils/getHermesCLIPath.ts

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+import os from 'os';
+
+/**
+ * Folder name of the Hermes compiler binary for the current OS.
+ */
+const getHermesOSBin = (): string => {
+  switch (os.platform()) {
+    case 'darwin':
+      return 'osx-bin';
+    case 'linux':
+      return 'linux64-bin';
+    case 'win32':
+      return 'win64-bin';
+    default:
+      throw new Error(
+        'OS not recognized. Please set hermesCLIPath to the path of a working Hermes compiler.'
```

Comment:
I have issues understanding this error message. The suggestion is to change hermesCLIPath but we only check on platform OS. Is this anyhow related?

Reply:
`hermesCLIPath` is an optional option for the plugin. By default the plugin tries to find the compiler on it's own

Reply:
yeah, but how is it related to `os.platform` failing to match supported dev platforms?

Reply:
I suppose we could move the error to the `getHermesCLIPath` function below